app/code/Magento/InventoryCatalog/view/adminhtml/templates/stocks.phtml

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
Where this template would be used? You already declared component for this.

Reply:
I'm sorry for that. I missed that when I was looking for unused code) I will delete it